userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -65,52 +65,55 @@ To know more, follow our https://docs.killbill.io/latest/userguide_deployment.ht
 
 === Kill Bill Platform
 
-We refer to Kill Bill as a *platform* because of its extensibility: You can write plugins to **modify the behavior of the system** or to **integrate with third-party vendors**, such as fraud detection solutions (Accertify, Feedzai, etc.) or Tax providers (Avalara AvaTax, etc.). We offer some open-source plugins on our http://github.com/killbill/killbill[GitHub organization], but in order to match your exact business requirements you may need to write your own.
+We refer to Kill Bill as a *platform* because of its extensibility: You can write plugins to **modify the behavior of the system** or to **integrate with third-party vendors**, such as fraud detection solutions (Accertify, Feedzai, etc.) or Tax providers (Avalara AvaTax, Vertex, etc.). We offer some open-source plugins on our http://github.com/killbill/killbill[GitHub organization], but in order to match your exact business requirements, you may need to write your own.
 
-
-As an example, let's imagine a Kill Bill deployment with two plugins, the first one a tax plugin that will interact with a 3rd party tax provider, and the second one, a payment plugin that will interact with a payment gateway.
+As an example, let's imagine a Kill Bill deployment with two plugins, the first one a tax plugin that will interact with a 3rd party tax provider, and the second one, a payment plugin, that will interact with a payment gateway.
 
 {empty} +
 
 [caption=""]
-//.Kill Bill: Big Picture
+//.Kill Bill: Deployment With Two Plugins
 image::../assets/what_is_killbill/KillBill_Plugins.png[format=png,align=center,title-align=center]
 
+These plugins are bundled as JAR files et deployed in the same Tomcat environment as Kill Bill. They can access the full spectrum of Kill Bill APIs, expose custom HTTP endpoints, or even access the database if they need to persist custom state.
+
+Any customization of Kill Bill will often require such plugins to be written. As such, it is not uncommon for deployments with custom business processes to have a custom of these, in addition to _connector_ plugins, which integrate with third-party systems.
```

Comment:
Not sure what you mean by "...to have a **custom** of these."

Reply:
I meant _a couple_.

Now, maybe I need to find a synonym for _custom_...

![image](https://user-images.githubusercontent.com/51940/153062004-73299577-3338-4b5c-9bbd-59f44c22b33f.png)